modules/mono/editor/bindings_generator.cpp

```diff
@@ -1489,7 +1489,6 @@ Error BindingsGenerator::generate_glue(const String &p_output_dir) {
 	}
 
 	if (tools_sequence) {
-		tools_sequence = false;
```

Comment:
Same situation as for the offsets above, there seems to be a pattern of `if (tools_sequence) { tools_sequence = false; }` here, each time an `#endif` is added. If more code is added below expecting the same pattern, it might break if the contributor does not make sure that the pattern was properly repeated the last time.

CC @neikeq

Reply:
Yes, I would keep this assignment for now. In the future I may move that to functions.

Reply:
Will do.